examples/hello_milvus.py

```diff
@@ -77,9 +77,7 @@ def hello_milvus():
 
     # show result
     for hits in res:
-        for hit in hits:
-            # Get value of the random value field for search result
-            print(hit, hit.entity.get("random_value"))
+        print("OK! search results: ", hits[1])
```

Comment:
Recommend to roll it back.

Reply:
ok, i will roll it back. that's only my test